### ISSUE TITLE
References

### ISSUE THREAD

**Issue by Johswald** [mariogeiger/gradientflow#1]
Hi Mario! Thanks a lot for this repo. 

Can you point me towards references that analyse how well your approximations compared to gradient flow? I am talking about your "compare method" leading to learning rates being de/increased. 
Intuitively these rules make sense but I would like to study this a bit more. 

Thank you very much!

**Comment by mariogeiger** [comment]
Hi, this is a legitimate question. The only measures I have is one plot in appendix G of [1906.08034](https://arxiv.org/pdf/1906.08034.pdf)

**Comment by mariogeiger** [comment]
One alternative to this library that I need to consider i.e. make proper numerical comparison is Adaptive stepsize (Dormand-Prince) Runge-Kutta.

wikipedia:
> Dormand–Prince is currently the default method in the ode45 solver for MATLAB and GNU Octave and is the default choice for the Simulink's model explorer solver. It is an option in SciPy's ODE integration library.[1] TensorFlow,[2] Fortran,[3] Java, [4] C++, [5] and Jax, [6] implementations are also available.

It has a jax implementation  [jax.experimental.odeint](https://github.com/google/jax/blob/main/jax/experimental/ode.py#L145) therefore it can easily be used to minimize neural networks losses.

**Comment by Johswald** [comment]
Thanks! I was more hoping for mathematical insights why your method does approximate the gradient flow well but I will also try Dormand–Prince then. Thanks anyway
